workers/unity/Assets/Playground/Scripts/MonoBehaviours/ToggleRotationCommandSender.cs

```diff
@@ -25,7 +25,7 @@ private void OnEnable()
         {
             if (obj.StatusCode != StatusCode.Success)
             {
-                GetComponent<SpatialOSComponent>().LogDispatcher.HandleLog(LogType.Error,
+                GetComponent<SpatialOSComponent>().Worker.LogDispatcher.HandleLog(LogType.Error,
```

Comment:
General comment: Logging is getting pretty verbose.

Reply:
I'm tempted to propose injecting the logger but I realise I am spending a lot of time with injectables so I am unsure if I'm making it a "when you have a hammer everything looks like a nail" moment.

How about adding logging methods to the spatialos component? Like, same signature as `Debug.Log` but `mySpatialOSComponent.Log` / `LogError`

Reply:
the idea behind adding the logger to the worker was to be able to directly identify which worker is logging. Though there have already been discussions going on to refactor this

Reply:
Yes having a simplified / not verbose logger API could be something for a next milestone

Reply:
There are a couple of soft plans for the logger. Injecting it doesn't seem insane in monobehaviours. But yes, not for this PR.